app/controllers/document_list_export_request_controller.rb

```diff
@@ -15,12 +15,17 @@ def show
   end
 
   def get_csv_file_from_s3(filename)
-    connection = Fog::Storage.new(
-      provider: "AWS",
+    params = if ENV.key? "AWS_ACCESS_KEY_ID"
+               { aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+                 aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"] }
+             else
+               { use_instance_profile: true }
```

Comment:
Where did this field come from?

Via [this issue](https://github.com/fog/fog-aws/issues/423), it looks like what you want is [use_iam_profile](https://github.com/fog/fog-aws/search?q=use_iam_profile&unscoped_q=use_iam_profile) (I can't see [use_instance_profile](https://github.com/fog/fog-aws/search?q=use_instance_profile&unscoped_q=use_instance_profile) anywhere)

Reply:
I found it on stackoverflow but upon doublechecking it seems like it was `use_iam_profile` there too, oops.

The conditional still seems to be necessary though, which is annoying.